lib/utils/text_strings.dart

```diff
@@ -340,6 +340,8 @@ class TextStrings {
   static const uploadFile = 'Please upload file first.';
   static const deleteFileConfirmationMsg =
       'Are you sure you want to remove this item from your transfer history?';
+  static const deleteFileConfirmationMsg_myFiles =
+      'Are you sure you want to remove this item from your files list?';
```

Comment:
This is not following the variable name convention.
Can we change `deleteFileConfirmationMsg_myFiles ` to camelCase ?

Reply:
yes we can.